entry_types/scrolled/package/src/frontend/index.js

```diff
@@ -7,6 +7,9 @@ import Entry from './Entry';
 import './global.module.css';
 import {EntryStateProvider} from '../entryState';
 
+import i18n from 'i18n-js';
+window.I18n = i18n;
+
```

Comment:
The [latest changes on master](https://github.com/codevise/pageflow/blob/108c2269343792cb630e199d07ab8ff29de113c9/entry_types/scrolled/app/helpers/pageflow_scrolled/entry_json_seed_helper.rb#L18) introduce a single global function called   `pageflowScrolledRender`. If we use this function to pass the public translations, we don not need to export the `I18n` object on window.